app/containers/Questions/QuestionsHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import createdHistory from 'createdHistory';
+
+import AuthenticatedButton from 'containers/AuthenticatedButton';
+import messages from './messages';
+
+const askQuestion = () => createdHistory.push('questions/ask');
```

Comment:
Let's do a minor refactoring for routes. I think it will be good to create a file with constants for all of our routes. In case if route urls change, we can update them in a single place.